### PR TITLE
Feature 1371 blocking wr

### PR DESCRIPTION
**Pull request by CPKalb**
This is the associated METplus pull request to the METcalcpy pr you did a week or two ago.  The blocking and weather regime calculations have been relocated to METcalcpy instead of living in the use case section of METplus

- [ ] Describe testing already performed for these changes:</br>
Ran the use cases to verify that they worked UserScript_fcstGFS_obsERA_Blocking.conf and UserScript_fcstGFS_obsERA_WeatherRegime.conf

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
This was previously tested for METcalcpy so I don't know that more testing is required.  But, if you want, check out this feature_1371, METcalcpy, and METplotpy and run the use cases: UserScript_fcstGFS_obsERA_Blocking.conf and UserScript_fcstGFS_obsERA_WeatherRegime.conf

- [ ] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes or No]**
Yes

- [ ] Do these changes include sufficient testing updates? **[Yes or No]**
Yes

- [ ] Will this PR result in changes to the test suite? **[Yes or No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>
No

- [ ] Please complete this pull request review by **[Fill in date]**.</br>
Before next release

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the PR definition above.
- [ ] Ensure the PR title matches the feature or bugfix branch name.
- [ ] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Organization** level software support **Project** or **Repository** level development cycle **Project**
Select: **Milestone** as the version that will include these changes
- [ ] After submitting the PR, select **Linked issues** with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

**Comment by CPKalb** [comment]
It looks like we need to add a couple of files for truth, as the difference comparison failed because they do not exist

File: .github/parm/use_case_groups.json [hunk 1]
```diff
@@ -122,7 +122,7 @@
   {
     "category": "s2s",
     "index_list": "1-3",
-    "run": false
+    "run": true
```
Comment (georgemccabe) [inline .github/parm/use_case_groups.json:125]: Turn off use case for pull request so other feature branches don't run these cases.
```suggestion
    "run": false
```

Comment (georgemccabe) [inline .github/parm/use_case_groups.json:125]: Turn off use cases for pull request so they aren't run on push for future pull requests
```suggestion
    "run": false
```

File: .github/parm/use_case_groups.json [hunk 2]
```diff
@@ -152,7 +152,7 @@
   {
     "category": "s2s",
     "index_list": "11",
-    "run": false
+    "run": true
```
Comment (georgemccabe) [inline .github/parm/use_case_groups.json:155]: Same as above
```suggestion
    "run": false
```

**Review by georgemccabe — APPROVED** [review]
The changes are straight forward and look good. A lot of files were removed and moved to METcalcpy. The automated tests ran successfully.

The use case groups should be turned off (run = false) before the pull request so that they do not run on a push event for future feature branches. I will make these changes.

One use case generated some additional output files, so we will need to update the develop-ref branch. I will take care of this step.

I approve.

**Pull request merged**